vkcs/compute/data_source_flavor.go

```diff
@@ -102,6 +105,13 @@ func DataSourceComputeFlavor() *schema.Resource {
 				Description: "The flavor visibility.",
 			},
 
+			"cpu_generation": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The `cpu_generation` of the flavor.",
```

Comment:
lets add a link https://cloud.vk.com/home/node/app/docs/base/iaas/concepts/vm-concept#cpu_generations_a045e625

we will ask nova owners to add cpu_generation values there

Reply:
Add a note, check please that it looks the way you wanted